### PR TITLE
feat(languages): add zsh and bash support

### PR DESCRIPTION
**Pull request by Luke-zhang-04**
Add support for zsh and bash
- #274
- #273
- #272

### CODE REVIEW

**Comment by Luke-zhang-04** [comment]
I'm gonna see if I can combine sh, zsh, and bash stats together into 1 number

File: src/onefetch/info.rs [hunk 1]
```diff
@@ -131,14 +132,41 @@ impl std::fmt::Display for Info {
...
+                        let mut languages = deduped_iter.by_ref().take(6).collect::<Vec<_>>();
+                        let other_sum = deduped_iter.fold(0.0, |acc, x| acc + x.1);
                         languages.push(("Other".to_owned(), other_sum));
                         languages
                     } else {
-                        iter.collect()
+                        // HELP WANTED can't sort by percentage
+                        let languages = deduped_iter.collect::<Vec<_>>();
+                        languages
                     }
```
Comment (Luke-zhang-04) [inline src/onefetch/info.rs:170]: I can't seem to figure out how to sort the stats by percentage. I've never touched Rust so I have no idea what to do

**Comment by o2sh** [comment]
Hi @Luke-zhang-04, sorry for the late answer.

> I'm gonna see if I can combine sh, zsh, and bash stats together into 1 number.

We don't want that. Showing multiple entries for sh, zsh, and bash provide the user with more insightful information.

Concerning your first commit, adding a line in the define_language! macro is not sufficient to add support for a language: you also need to provide an ASCII logo. Please read [How to add support for a new language](https://github.com/o2sh/onefetch/blob/master/CONTRIBUTING.md#adding-support-for-a-new-language)

**Comment by Luke-zhang-04** [comment]
Ok, I'll do that instead then. I was thinking about combining them because that's how GitHub does it.

**Comment by Luke-zhang-04** [comment]
Bash:
image

Z-shell:
image

**Comment by o2sh** [comment]
LGTM 👍 
Thanks @Luke-zhang-04

**Pull request merged**